### PR TITLE
fix: update samtools_merge

### PR DESCRIPTION
**Pull request by maxulysse**
Just don't want the merged file to be have the `_merged` suffix by default.
That can go to conf/modules.config

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `<SOFTWARE>.version.txt` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: modules/samtools/merge/main.nf [hunk 1]
```diff
@@ -22,14 +22,14 @@ process SAMTOOLS_MERGE {
     tuple val(meta), path(bams)
 
     output:
-    tuple val(meta), path("*merged.bam"), emit: merged_bam
-    path  "*.version.txt"              , emit: version
+    tuple val(meta), path("${prefix}.bam"), emit: merged_bam
```
Comment (drpatelh) [inline modules/samtools/merge/main.nf:25]:
```suggestion
    tuple val(meta), path("${prefix}.bam"), emit: bam
```

Comment (drpatelh) [reply modules/samtools/merge/main.nf:25]: May need to change the `output` options in `meta.yml` too

**Review by drpatelh — APPROVED** [review]
LGTM! Minor change :)

File: modules/samtools/merge/main.nf (hunk 1 above)
Comment (maxulysse) [reply modules/samtools/merge/main.nf]: I think it should be fine that way

**Pull request merged**